Buildfile

```diff
@@ -7,6 +7,6 @@ cookbook.depends 'propsd' do |propsd|
   propsd.path './cookbook'
 end
 
-profile :default do |default|
-  default.chef.run_list 'propsd::default'
+profile :default do |test|
```

Comment:
Instead of doing this you should add a new `:test` profile that extends `:default`. For testing you can do `build local test` and the default profile is preserved.